### ISSUE TITLE
Possible CSS incompatibility with WP5.4 post admin area 

### ISSUE THREAD

**Issue by Tanmccuin** [saltnpixels/ignition#10]
After upgrading on a fresh install of ignition i've noticed strange resizing issues with the sidebar in the page/post editor - where the sidebar when visible, causes the editor area to reduce in size by half and only occupy half of the visible screen. 

Digging into the CSS i saw this bit of CSS in the Gutenberg SASS

`@media (min-width: 768px)
#wpwrap .edit-post-layout.is-sidebar-opened .edit-post-sidebar {
    width: 30%;
}`

disabling that bit of CSS in the gutenberg-editor-styles.scss seems to fix the issue.

**Comment by saltnpixels** [comment]
Yes! this has been addressed in the latest ignition build! Thank you.
Gutenberg keeps changing their styles and code. So we have to keep changing ours. 

Our gutenberg styles are now put in two files. gutenberg-layout and gutenberg-styles. Put your styles in gutenberg-styles. This way you can keep updating the layout one easily with a newer version.

**Comment by Tanmccuin** [comment]
Awesome - glad to hear it. Interesting continuation, i grabbed the latest versions of the relevant sass files and even on upload (using wp-sass for post go-live compilation) they don't appear to be recognized and recompiled like other sass files (which otherwise work great).  Also - side though, is there a prefered method of editing sass to avoid conflicts and ensure upgradeability of the core theme? Child theme? specific SASS files?

**Comment by saltnpixels** [comment]
hmmm running just sass might not work without npm and gulp anymore because we make a template-parts.scss file and then delete it using gulp. 

It contains all the sass from the template-parts folder. We concat all those and then put it into template parts, then run sass.
